picard/file.py

```diff
@@ -158,6 +158,21 @@ def _load(self, filename):
         """Load metadata from the file."""
         raise NotImplementedError
 
+    def _test_same(self):
+        if config.setting['clear_existing_tags']:
+            return False
+        new_metadata = self.new_metadata
+        orig_metadata = self.orig_metadata
+        tags = set(new_metadata.keys() + orig_metadata.keys())
+        for name in filter(lambda x: not x.startswith("~") and self.supports_tag(x), tags):
+            new_values = new_metadata.getall(name)
+            orig_values = orig_metadata.getall(name)
+            if new_values != orig_values:
+                return False
+        if orig_metadata.length != new_metadata.length:
```

Comment:
I think this is not correct, see #109 and [PICARD-342](https://tickets.metabrainz.org/browse/PICARD-432?jql=project%20%3D%20PICARD%20AND%20text%20~%20%22length%22).